src/builders/SceneBuilder.ts

```diff
@@ -1,16 +1,65 @@
 import { RocketAssetManager, RocketConfig, TextureFormat } from "rl-loadout-lib"
-import { Cache, LoadingManager, Scene } from "three"
+import { Cache, Group, LoadingManager, Scene } from "three"
 import { DRACOLoader } from "three/examples/jsm/loaders/DRACOLoader"
 import { GLTFLoader } from "three/examples/jsm/loaders/GLTFLoader"
 
-import GameFieldAssets from "../loaders/scenes/GameFieldAssets"
-import { loadRlLoadout } from "../loaders/storage/loadRlLoadout"
+import { AssetLoader } from "../loaders/AssetLoader"
+import HighQualityAssetLoader from "../loaders/impl/HighQualityAssetLoader"
+import LowQualityAssetLoader from "../loaders/impl/LowQualityAssetLoader"
+import MediumQualityAssetLoader from "../loaders/impl/MediumQualityAssetLoader"
+import PlayerManager from "../managers/models/PlayerManager"
+import QualityManager, { QualityOptions } from "../managers/QualityManager"
 import SceneManager from "../managers/SceneManager"
 import { ExtendedPlayer } from "../models/ReplayMetadata"
 import { buildBall } from "./ball/buildBall"
 import { buildPlayfield } from "./field/buildPlayfield"
-import { buildRocketLoadoutGroup } from "./player/buildRocketLoadoutScene"
+import { generateSprite } from "./player/generateSprite"
 import { addLighting } from "./scene/addLighting"
+import { getGroupName } from "./utils/playerNameGetters"
+
+const buildAssetManager = (loadingManager?: LoadingManager) => {
+  const gltfLoader = new GLTFLoader(loadingManager)
+  const dracoLoader = new DRACOLoader()
+  dracoLoader.setDecoderPath("/draco/")
+  gltfLoader.setDRACOLoader(dracoLoader)
+
+  const config = new RocketConfig({
+    gltfLoader,
+    loadingManager,
+    textureFormat: TextureFormat.PNG,
```

Comment:
for medium you can use `textureQuality: TextureQuality.LOW`, that will load lower resolution png files

Reply:
Is this supported in a newer version? I'm not seeing support for it yet